frontera/contrib/backends/redis_backend/__init__.py

```diff
@@ -105,7 +129,21 @@ def get_next_requests(self, max_n_requests, partition_id, **kwargs):
                 request.meta[FIELD_SCORE] = score
                 results.append(request)
         if len(to_remove) > 0:
-            connection.zrem(partition_id, *to_remove)
+            timeout = _get_retry_timeouts()
+            while True:
+                try:
+                    connection = StrictRedis(connection_pool=self._pool)
```

Comment:
would it make more sense to wrap StrictRedis class with methods proxying to original StrictRedis with while/try/except ? This would help to avoid repetitions.